### PR TITLE
Add recipe for perl-date-simple

### PR DESCRIPTION
**Pull request by deeuu**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by deeuu** [comment]
Looks like the Windows build failed. 

From the [build log](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=72845&view=logs&jobId=240f1fee-52bc-5498-a14a-8361bde76ba0&taskId=23010c01-6a9d-5f94-9037-792618644b77&lineStart=633&lineEnd=640&colStart=1&colEnd=46):
```
Have \minico~1\conda-~2\perl-d~2\_h_env\library\lib
Want \strawb~1\perl\lib
Your perl and your Config.pm seem to have different ideas about the
architecture they are running on.
Perl thinks: [lib]
Config says: [MSWin32-x64-multi-thread]
This may or may not cause problems. Please check your installation of perl
if you have problems building this extension.
```

@conda-forge/help-perl Can any one help with this?

Would it be possible to skip Windows if this isn't an easy fix?

File: recipes/perl-date-simple/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  imports:
+    - Date::Simple
+    - Date::Simple::D8
+    - Date::Simple::Fmt
+    - Date::Simple::ISO
+
+about:
+  home: http://metacpan.org/pod/Date::Simple
+  license: GPL-2.0
+  license_file: COPYING
```
Comment (scopatz) [inline recipes/perl-date-simple/meta.yaml]: `license_family: GPL` should be added

**Comment by scopatz** [comment]
> Would it be possible to skip Windows if this isn't an easy fix?

Yep! Add:

```yaml
build:
  skip: True  # [win]
```

**Comment by deeuu** [comment]
[Apparently](https://github.com/ContinuumIO/anaconda-issues/issues/10690) there are issues with the default perl having the compiler hard-coded in its config files. Tried using `perl=5.26.2=h470a237_0` as suggested, but the compilation failes due to missing `xlocale.h`, which [might only be resolved](https://github.com/conda-forge/perl-feedstock/issues/28) by compiling against glibc 2.26.

Given that this module is highly outdated, and I'm no Perl enthusiast, I'm going to close this.

**Pull request closed without merging**